lib/thrift/muzic/src/defl_static.c

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
please run `git mv lib/thrift/muzic/src/*.c lib/thrift/src` and `mkdir lib/thrift/include/muzic; git mv lib/thrift/muzic/include/*.h lib/thrift/include/muzic`. 

In relevant source files, you could probably adjust e.g. `#include "uzlib.h"` to `#include <muzic/uzlib.h>`.